Fix build on x86_64 with -Werror - not sure those functions require "long"
(otherwise they are broken on x86_64), but changing this would break the API.

--- src/indent.cc~	2010-10-14 13:57:11.000000000 +0200
+++ src/indent.cc	2011-10-08 16:40:04.000000000 +0200
@@ -26,7 +26,7 @@
   inline long int& indent (std::ostream& o)
   {
     // The slot to store the current indentation level.
-    static const long int indent_index = std::ios::xalloc ();
+    static const int indent_index = std::ios::xalloc ();
     return o.iword (indent_index);
   }
 
@@ -54,7 +54,7 @@
     o << std::endl;
     // Be sure to be able to restore the stream flags.
     char fill = o.fill (' ');
-    return o << std::setw (indent (o))
+    return o << std::setw ((int)indent (o))
 	     << ""
 	     << std::setfill (fill);
   }
